.github/workflows/test.yml

```
@@ -30,7 +30,7 @@ jobs:
   pwa-kit:
     strategy:
       matrix:
-        node: [14]
+        node: [14, 16]
         npm: [6, 7, 8]
```

Same. Node 16 requires npm7. Let's skip the Node 16 and npm 6 build from the matrix:
```suggestion
        npm: [6, 7, 8]
        exclude:
          - node: 16
            npm: 6
```

---

Ah I see what you mean - node 16 comes with npm 7 or 8, so there's no point testing node 16 with npm 6. In that case, would we also exclude node 14 with npm 7 and 8?

https://nodejs.org/en/download/releases/

---

Ok, let me think a bit more. 
Node 16 comes by default with npm 7.
Support to npm@6 will be until Node 14 goes EOL because npm@6 is shipped with Node 14.

I'm wrong. We still need to test npm 6 with both Node 14 and Node 16 until Node 14 reaches EOL.

---

Ok, it makes sense that we need to support npm 6 until node 14 reaches EOL. I wonder whether in practice, anyone would use a different version of npm than the one that comes installed with their current node version though 🤔 

I will change it back, but something interesting I noticed is that after Brian and I fixed the tests that were consistently failing, several more builds have failed and (for the most part) they happen on the non-default node/npm combinations (ex. [here](https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/4206121998/jobs/7299170597), [here](https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/4205122792/jobs/7296753084), and [here](https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/4199789414/jobs/7285094164)).